### DISCUSSION TITLE
Field creationOptions for Group api

### DISCUSSION THREAD

**Discussion by ayblanchet** [microsoftgraph/msgraph-sdk-dotnet#2342] [Q&A]
Hi,
Could you say me how to get creationOptions field from Group api? I have no "creationOptions" property on group object and it's not present in additionalData.

I use Microsoft.graph 5.4.0.

Thank you

**Comment by andrueastman** [comment]
Any chance you can confirm where the property is coming from? Taking a look at the documentation, the Group object does not have the property existing. 

https://learn.microsoft.com/en-us/graph/api/resources/group?view=graph-rest-1.0#properties

**Comment by ayblanchet** [comment]
It's possible to call: https://graph.microsoft.com/v1.0/groups in https://developer.microsoft.com/en-us/graph/graph-explorer. 
The properties is available (see screenshot)
![Graph-group](https://github.com/microsoftgraph/msgraph-sdk-dotnet/assets/17802241/08cb2d87-d08e-432f-ba8d-724979c98b16)

**Reply by andrueastman** [reply]
Any chance you are able to see the property in the SDK when you add it as a select parameter? I believe it should be something like this

```cs
var result = await graphClient.Groups.GetAsync((requestConfiguration) =>
{
	requestConfiguration.QueryParameters.Select = new string []{ "creationOptions" };
});
```

**Reply by ayblanchet** [reply]
I see I used "beta" entry. It's work on v1.0. CreationOption will be disappear? If yes, will it be replace by another field?
Thank you,

**Reply by andrueastman** [reply]
If the property is to be deprecated, the API owners would ideally communicate that in advance by adding an entry to the Graph Changelog. https://developer.microsoft.com/en-us/graph/changelog/?filterBy=Deprecation,v1.0

However, as that is more of an api side question, you can ask the same at the link below to get more clarification/guidance from the API owners. https://developer.microsoft.com/en-us/graph/support
